galleries/examples/subplots_axes_and_figures/twinx_align.py

```diff
@@ -0,0 +1,95 @@
+"""
+===============================================
+Let twin-axis aligned at the specified position
+===============================================
+
+Let the left-axis and right-axis aligned at the specified position.
+
+In some data that need twin-axis plot, a point at left-axis and
+another point at right-axis have same meaning, they shoud be aligned.
+For example, we plot netvalue curve of a portfolio at twin-left-axis
+and profit raito curve at twin-right-axis, the point 1.0 at the
+left-axis and the point 0.0 at the right-axis both mean the begin
+state of the portifolio, so they should be aligned.
+"""
+import pandas as pd
+from matplotlib import pyplot as plt
+
+
+# Sim data
+net_in = [100, 0, 0, 0, 20, 0, 0, 0, 0, -20, 
+          0, 0, 30, 0, 0, 0, 0, 0, -30, 0]
+net_gain= [0, -2, -3, 5, 2, 3, 4, 5, 5, -1,
+           -4, -10, 2, 5, 9, 6, 0, 1, -1, 9]
+df = pd.DataFrame({"net_in": net_in, "net_gain": net_gain})
```

Comment:
Can this be done in numpy instead of pandas?

Reply:
Yes, I have already done this with numpy.